docs/nx-cloud/concepts/building-blocks-fast-ci.md

```diff
@@ -0,0 +1,35 @@
+# Building Blocks of Fast CI
+
+Nx has many features that make your CI faster. Each of these features speeds up your CI in a different way, so that enabling an individual feature will have an immediate impact. These features are also designed to complement each other so that you can use them together to create a fully optimized CI pipeline.
+
+## Use Cutting Edge Build Tools
```

Comment:
I wouldn't use "cutting edge" but rather "fast". Cutting edge sounds unsafe, which is not necessarily what we want to community. But rather the fact that the build tooling you're using is your starting "budget". Can't really get faster than that.